testing/testmedia.cpp

```diff
@@ -890,6 +891,188 @@ void SrtCommon::PrepareClient()
 }
 
 #if ENABLE_EXPERIMENTAL_BONDING
+#if 0
+void SrtConn_ConnectCB(void *opaq, SRTSOCKET ns, int errorcode, const struct sockaddr *psaPeer, int token)
+{
+    char szEndPoint[SRTC_ENDPOINTLEN];
+    SrtConn *srtc = (SrtConn *)opaq;
+    int iLID = token-1;
+    int iRejectReason = SRT_REJ_UNKNOWN;
+    srtcLog(srtc, LOG_DEBUG, "connectCB(%p,%x,%d,%s,%d)\n",
```

Comment:
dead code

Reply:
Right. That's not even a dead code, rather a copy of the original code that was causing crashes in the library, which serves as a pattern to that one. I'll ship this off.